### ISSUE TITLE
getRawPacketReadOnly() const-correctness

### ISSUE THREAD

**Issue by luizfeldmann** [seladb/PcapPlusPlus#1573]
Hi,

It seems that both the [read-only](https://github.com/seladb/PcapPlusPlus/blob/4af4b8b04e6d5dd40b8e69cc30c5c1f62e7fe368/Packet%2B%2B/header/Packet.h#L181) and [non-read-only](https://github.com/seladb/PcapPlusPlus/blob/4af4b8b04e6d5dd40b8e69cc30c5c1f62e7fe368/Packet%2B%2B/header/Packet.h#L155) getters of the raw packet from the parsed packet have the exact same signature and behaviour:

```
RawPacket* getRawPacket() const;
RawPacket* getRawPacketReadOnly() const;
```

Wouldn't it be more correct to declare the read-only getter as returning a pointer-to-const, and similarly, the non-read-only method not be marked as const method as it can alter the underlying object ?

```
RawPacket* getRawPacket();
const RawPacket* getRawPacketReadOnly() const;
```

Best regards,

**Comment by egecetin** [comment]
@luizfeldmann Feel free to open a PR to fix it if you have time. Contributions are welcome!

**Comment by egecetin** [comment]
@luizfeldmann Thanks for reporting. Fixed with #1637
